pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -309,7 +310,7 @@ func (configgen *ConfigGeneratorImpl) buildSidecarListeners(env *model.Environme
 // buildSidecarInboundListeners creates listeners for the server-side (inbound)
 // configuration for co-located service proxyInstances.
 func (configgen *ConfigGeneratorImpl) buildSidecarInboundListeners(env *model.Environment, node *model.Proxy, push *model.PushContext,
-	proxyInstances []*model.ServiceInstance) []*xdsapi.Listener {
+	proxyInstances []*model.ServiceInstance, _ *model.SidecarScope) []*xdsapi.Listener {
```

Comment:
why add a no use param?

Reply:
Oh its temporary. I am going to implement this as well.